code/game/objects/structures/crates_lockers/closets/secure/personal.dm

```diff
@@ -53,4 +53,22 @@
 		else
 			user << "<span class='danger'>Access Denied.</span>"
 	else
-		return ..()
\ No newline at end of file
+		return ..()
+
+/obj/structure/closet/secure_closet/personal/verb/verb_resetlock()//personal locker ownership reset.
+	set src in oview(1)
+	set category = "Object"
+	set name = "Reset Lock"
+
+	if(opened)
+		if(src.broken)
+			usr << "<span class='danger'>It appears to be broken.</span>"
```

Comment:
Also you don't need to call `src` in in most of these var checks and such. `if(broken)` is the same as `if(src.broken)`, only faster.